src/imitation/scripts/train_adversarial.py

```diff
@@ -236,12 +237,27 @@ def should_plot_now(epoch) -> bool:
                                        trainer.env,
                                        n_episodes=n_episodes_eval)
     assert stats["n_traj"] >= n_episodes_eval
-    mean = stats["return_mean"]
-    std_err = stats["return_std"] / math.sqrt(n_episodes_eval)
-    print(f"[result] Mean Episode Return: {mean:.4g} ± {std_err:.3g} "
+    ep_reward_mean = stats["return_mean"]
+    ep_reward_std_err = stats["return_std"] / math.sqrt(n_episodes_eval)
+    print("[result] Mean Episode Return: "
+          f"{ep_reward_mean:.4g} ± {ep_reward_std_err:.3g} "
           f"(n={stats['n_traj']})")
 
-    return dict(mean=mean, std_err=std_err)
+    reward_path = os.path.join(log_dir, "checkpoints", "final", "discrim")
+    # TODO(shwang): I think Serializable should store the save_type, and
```

Comment:
This seems like a reasonable modification.

Reply:
Not necessary any more since "DiscrimNet" is now a valid key.